### DISCUSSION TITLE
How to stop job (DETECT OBJECTS)

### DISCUSSION THREAD

**Discussion by C8opmBM** [immich-app/immich#1896] [Q&A] [answered]
Hey
I've updated the machine learning image (before I used a custom image built for my noavx cpu - j5005), and I pressed the start job thinking it will just check, but restarted the job which is too much for my cpu (300% usage) and I have thousand of pics. 
I will upgrade in the summer, so I want to cancel the job, however I cannot.
I re-created the container, but it still picks up the job at creation.
Is there a way to cancel the job after it started (maybe a command to be run inside the container?).
Or do I need to disable the machine learning container?

Thanks!

**Comment by bo0tzz** [comment, marked as answer]
The jobs are queued via redis. If you delete and recreate that container, it should clear the queue.

**Comment by C8opmBM** [comment]
Deleted the stack and re-created it. Unfortunately, it picked up right from where it left. 
No biggie, I will keep the machine learning off for now until I figure it out.

LE: sorry, it did work. Grafana playing tricks on me, having it on 6 hour, didn't notice cpu work stopped. Thank you!
